Packs/FeedRecordedFuture/ReleaseNotes/1_2_0.json

```diff
@@ -0,0 +1,4 @@
+{
+    "breakingChanges": true,
+    "breakingChangesNotes": "Updated the CVE risk labels to be compatible with the Recorded Future documentation"
```

Comment:
This doesn't tell the customer what are the changes. It's best to detail the table changes.